security-insights-schema-1.0.0.yaml

```diff
@@ -130,6 +130,35 @@ properties:
                 $id: '#/properties/contribution-policy/properties/accepts-bot-pull-requests'
                 description: 'Define if the maintainers accept pull-requests generated by bots or automated tools.'
                 type: boolean
+            bots-list:
+                $id: '#/properties/contribution-policy/properties/bots-list'
+                description: 'List of allowed and denied bots. This property can overwrite `accepts-bot-pull-requests`.'
+                additionalItems: false
+                items:
+                    anyOf:
+                    -   $id: '#/properties/contribution-policy/properties/bots-list/items'
+                        additionalProperties: false
+                        properties:
+                            bot-name:
+                                $id: '#/properties/contribution-policy/items/anyOf/0/properties/bot-name'
+                                description: 'Bot name.'
+                                type: string
```

Comment:
Is there any sort of wildcarding we want to support here?

Reply:
e.g. `*securty*`, and this blocks or allows all bots that have `security` in the name?